buckaroo/RemoveCommand.fs

```diff
@@ -5,8 +5,8 @@ open Buckaroo.RichOutput
 
 let task (context : Tasks.TaskContext) (packages : List<PackageIdentifier>) = async {
   context.Console.Write(
-    (text "Removing [ ") + 
-    (packages |> Seq.map PackageIdentifier.showRich |> RichOutput.concat (text " ")) + 
+    (text "Removing [ ") +
+    (packages |> Seq.map PackageIdentifier.showRich |> RichOutput.concat (text " ")) +
```

Comment:
Pretty sure `RichOutput` has `+` operator implemented. So you could do things like: `"abc" |> text |> ((+) "def")`